Source/WebCore/page/FrameView.h

```diff
@@ -911,6 +913,7 @@ class FrameView final : public ScrollView {
     RefPtr<Node> m_nodeToDraw;
     std::optional<SimpleRange> m_pendingTextFragmentIndicatorRange;
     String m_pendingTextFragmentIndicatorText;
+    bool m_doNotResetFragmentTimer { false };
```

Comment:
I think this isn't just about the timer. It's "should incoming scrolling cancel maintaining scroll-to-text-fragment position". Try to make a better name. It's hard to compress down.